blueprints/ember-cli-typescript/index.js

```diff
@@ -45,8 +54,24 @@ module.exports = {
           paths[`${appName}/*`].push(`${addon}/app/*`);
         }
 
+        paths['*'] = ['types/*'];
+
         return JSON.stringify(paths, null, 2).replace(/\n/g, '\n    ');
       },
+      baseDeclarations: dasherizedName => {
+        const isDummyApp = dasherizedName === 'dummy';
+        const useAppDeclarations = !(isAddon || isDummyApp);
+        return useAppDeclarations ? APP_DECLARATIONS : '';
+      },
+    };
+  },
+
+  fileMapTokens(/*options*/) {
+    // Return custom tokens to be replaced in your files.
+    return {
+      __app_name__(options) {
+        return options.dasherizedModuleName;
```

Comment:
We want this to be `'dummy'` for addons, right? I think `dasherizedModuleName` will be the addon name here in that situation.

Reply:
This was the other bit I was wondering about, yeah.

Reply:
Updated to that effect, @dfreeman.